src/main/java/com/paneedah/mwc/groovyscript/script/CraftingStation.java

```diff
@@ -2,54 +2,60 @@
 
 import com.cleanroommc.groovyscript.api.GroovyLog;
 import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
 import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
 import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import com.paneedah.mwc.ProjectConstants;
 import com.paneedah.mwc.groovyscript.recipes.GSCrafting;
 import com.paneedah.weaponlib.crafting.CraftingEntry;
 import com.paneedah.weaponlib.crafting.CraftingGroup;
 import com.paneedah.weaponlib.crafting.CraftingRegistry;
 import com.paneedah.weaponlib.crafting.IModernCraftingRecipe;
+import net.minecraft.item.ItemStack;
 
 import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
 
 import static com.paneedah.mwc.ProjectConstants.NAME;
 import static com.paneedah.mwc.groovyscript.MWCGroovyPlugin.craftingStation;
-import static com.paneedah.weaponlib.crafting.CraftingGroup.GUN;
+import static com.paneedah.weaponlib.crafting.CraftingGroup.*;
 import static com.paneedah.weaponlib.crafting.CraftingRegistry.*;
 
+@RegistryDescription(linkGenerator = ProjectConstants.ID)
 public class CraftingStation extends VirtualizedRegistry<IModernCraftingRecipe> {
 
     @Override
     public void onReload() {
-        this.removeScripted().forEach(CraftingRegistry::deleteRecipeRegistry);
-        this.restoreFromBackup().forEach(CraftingRegistry::registerRecipe);
+        this.removeScripted().forEach(CraftingRegistry::registerRecipe);
+        this.restoreFromBackup().forEach(CraftingRegistry::deleteRecipeRegistry);
```

Comment:
This sounds wrong can you explain a bit what this does, like it sounds conflicting

Reply:
Yeah, this looks wrong. You are supposed to remove previously via script added recipes. And add back those which were removed